### ISSUE TITLE
Cannot run dagit

### ISSUE THREAD

**Issue by C-Loftus** [internetofwater/scheduler#11]
The docs for scheduler says to run `dagit -h ghost.lan -w workspace.yaml`. However, even after installing all requirements files it says the following:

It seems to be because of this:

https://github.com/dagster-io/dagster/discussions/18783

```sh
(.venv) cloftus@Coltons-MacBook-Pro ~/g/s/d/implnets (master)> dagit -h ghost.lan -w works
pace.yaml
Traceback (most recent call last):
  File "/Users/cloftus/github/scheduler/.venv/bin/dagit", line 5, in <module>
    from dagster_webserver.cli import main
  File "/Users/cloftus/github/scheduler/.venv/lib/python3.12/site-packages/dagster_webserver/__init__.py", line 1, in <module>
    from dagster._core.libraries import DagsterLibraryRegistry
  File "/Users/cloftus/github/scheduler/.venv/lib/python3.12/site-packages/dagster/__init__.py", line 84, in <module>
    from dagster._config.config_schema import ConfigSchema as ConfigSchema
  File "/Users/cloftus/github/scheduler/.venv/lib/python3.12/site-packages/dagster/_config/__init__.py", line 5, in <module>
    from .config_type import (
  File "/Users/cloftus/github/scheduler/.venv/lib/python3.12/site-packages/dagster/_config/config_type.py", line 6, in <module>
    from dagster._annotations import public
  File "/Users/cloftus/github/scheduler/.venv/lib/python3.12/site-packages/dagster/_annotations.py", line 9, in <module>
    from dagster._utils.backcompat import (
  File "/Users/cloftus/github/scheduler/.venv/lib/python3.12/site-packages/dagster/_utils/__init__.py", line 50, in <module>
    import dagster._seven as seven
  File "/Users/cloftus/github/scheduler/.venv/lib/python3.12/site-packages/dagster/_seven/__init__.py", line 16, in <module>
    from .compat.pendulum import PendulumDateTime as PendulumDateTime  # re-exported
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/cloftus/github/scheduler/.venv/lib/python3.12/site-packages/dagster/_seven/compat/pendulum.py", line 31, in <module>
    pendulum.DateTime if _IS_PENDULUM_2 else pendulum.Pendulum  # type: ignore[attr-defined]
                                             ^^^^^^^^^^^^^^^^^
AttributeError: module 'pendulum' has no attribute 'Pendulum'. Did you mean: '_pendulum'?
(.venv) cloftus@Coltons-MacBook-Pro ~/g/s/d/implnets (master) [0|1]> 
```

**Comment by C-Loftus** [comment]
seems to have dependency issues when trying to fix this

```
        File "/private/var/folders/ff/zdlbl2jn4jx2h5qfdzjrdqvc0000gn/T/pip-build-env-rd7l2wa7/overlay/lib/python3.12/site-packages/poetry/core/masonry/builders/wheel.py", line 262, in _run_build_script
          subprocess.check_call([self.executable.as_posix(), build_script])
        File "/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/subprocess.py", line 413, in check_call
          raise CalledProcessError(retcode, cmd)
      subprocess.CalledProcessError: Command '['/Users/cloftus/github/scheduler/.venv/bin/python', 'build.py']' returned non-zero exit status 1.
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for pendulum
Failed to build pendulum
ERROR: Could not build wheels for pendulum, which is required to install pyproject.toml-based projects

[notice] A new release of pip is available: 24.0 -> 24.2
[notice] To update, run: pip install --upgrade pip
```

**Comment by C-Loftus** [comment]
we no longer run dagit but instead use dagster. dagit appears deprecated
